source/ShopPanel.cpp

```diff
@@ -141,6 +142,14 @@ void ShopPanel::Draw()
 	DrawButtons();
 	DrawKey();
 
+	// Draw the Find button. Note: buttonZones are cleared in DrawButtons.
+	const Point findCenter = Screen::BottomRight() - Point(580, 20);
+	const Sprite *findIcon =
+		hoverButton == 'f' ? SpriteSet::Get("ui/find selected") : SpriteSet::Get("ui/find unselected");
```

Comment:
```suggestion
	const Sprite *findIcon = SpriteSet::Get(hoverButton == 'f' ? "ui/find selected" : "ui/find unselected");
```
now it actually fits on a single line